### ISSUE TITLE
Make some ObjC interfaces private

### ISSUE THREAD

**Issue by hvge** [lime-company/powerauth-mobile-sdk#24]
Current IOS  SDK is exposing too much implementation details in ObjC interfaces. We should move several classes (for example networking objects) into private folders, which will not be copied at the end of pod build script. 

So, the goal of this task is to simplify the public interface as much as possible.

**Comment by hvge** [comment]
Changes made in our interfaces:
- Removed `PA2PersonalizedEncryptedObject`. That class was not used in our source codes and this kind of E2EE will be replaced with new ECIES encryptor in future versions of SDK
- Removed `"PA2Networking.h"` header. We're now two separate headers as a replacement:
  - `PA2PublicNetworking.h` which imports all public networking classes except `PA2Client.h`
  - `PA2PrivateNetworking.h` which imports all private interfaces and declares PA2Client's private methods.

On top of that, following classes are now private:
- `PA2ActivationStatusRequest`
- `PA2CreateActivationRequest`
- `PA2DirectCreateActivationRequest`
- `PA2RemoveTokenRequest`
- `PA2EncryptedRequest` (new ECIES, should not affect existing codes)
- `PA2ActivationStatusResponse`
- `PA2CreateActivationResponse`
- `PA2VaultUnlockResponse`
- `PA2GetTokenResponse`
- `PA2EncryptedResponse` (new ECIES)

In short, basically all request & response model classes are now private. Currently only E2EE related model classes are still public, because that interface may be still in use. 


*Note that I may update this commend with additional explanations*
